Buildings/Resources/Scripts/Dymola/Controls/OBC/FDE/PackagedRTUs/DDSPsetSimulation.mos

```diff
@@ -0,0 +1,34 @@
+// Script generated by Dymola Mon Jul  6 16:21:59 2020
```

Comment:
This `.mos` file (and also other `.mos` files except `OperatingMode.mos`) did not specify simulating which models. 

Please look into file `Buildings/Resources/Scripts/Dymola/Controls/OBC/FDE/PackagedRTUs/Validation/OperatingMode.mos`. And also check in file `Buildings/Controls/OBC/FDE/PackagedRTUs/Validation/OperatingMode.mo`, following line:
``` 
  __Dymola_Commands(file="modelica://Buildings/Resources/Scripts/Dymola/Controls/OBC/FDE/PackagedRTUs/Validation/OperatingMode.mos"
```

Reply:
Right, all of the extra '.mos' files need to be deleted...